js/app/controllers/calcontroller.js

```diff
@@ -295,6 +295,28 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 							});
 						}
 					}
+
+					// TODO(leon): Trigger event instead
```

Comment:
do the event?

Reply:
The Calendar app currently doesn't support to be extended from other apps, that's why the integration was made directly into this app.